### ISSUE TITLE
Material Design NavigationView Style possible mismatch  

### ISSUE THREAD

**Issue by iurycarlos** [unoplatform/Uno.Themes#980]
## Current behavior
It seams we have a mismatch between MaterialNavigationViewStyle and Material Design.

Following the https://m2.material.io/components/navigation-drawer#anatomy , we should be able to add an image in the Header

However, just adding an image to the Header, this is the result we have:

![image](https://user-images.githubusercontent.com/6876973/236890850-5690a7d2-594c-4683-8edb-0159257c7ee8.png)

Of course a template there would do the job, but maybe would be interesting to have any sample about it to make it easy to the user.

## Expected behavior

Should be closer the look we have in Material Design docs.

## How to reproduce it (as minimally and precisely as possible)

Added the Xaml code at https://playground.platform.uno/#20aaff5a 
(adapted to Playground, of course)

## Environment

 

Nuget Package:

Package Version(s):
Uno 4.8.15
Uno.Material 2.5.3

**Comment by iurycarlos** [comment]
@Xiaoy312 didn't have opportunity to add treegraph in the plugin. Will paste here as soon as possible.

**Comment by Xiaoy312** [comment]
You need to use the [`PaneHeader`](https://learn.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.navigationview.paneheader?view=winui-2.8) property instead of [`Header`](https://learn.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.navigationview.header?view=winui-2.8), if you wish to add header to the pane (menu items) area.

Unfortunately the pane background property is not customizable, within the control api contract, which can only be done with lightweight styling (which im not sure if it is properly supported here, yet?).

Between header sections, it appears that you could generate a NavigationViewItemSeparator.

**Comment by iurycarlos** [comment]
Yeah. Seams pane background is really not supported. Using PaneHeader got the following result, that is still not what we could expect for Material Design:

![image](https://github.com/unoplatform/Uno.Themes/assets/6876973/233c8c9e-6f4c-4df4-9ca1-a123cd783125)

In resume: 
- Background is not customizable / supported
- Material Design Header is different from PaneHeader or Header.

**Comment by iurycarlos** [comment]
About backgrounds... using the this code, https://playground.platform.uno/#5a4d98cc 
in the Xaml Studio, overriding some resources we get the following result: 

![image](https://github.com/unoplatform/Uno.Themes/assets/6876973/3a784b7b-dd71-4357-8476-ed46514f603b)

**Comment by iurycarlos** [comment]
@NVLudwig  
FYI

**Comment by NVLudwig** [comment]
Looking better :)
1. @iurycarlos are we support Toggle + Image (I have Toggle OR Image in Figma)?
2. @iurycarlos whats are your thoughts about support of a "custom" area (between the header and the Navigaion Sections) where users could swap in other components like TextBox (for Search), Switch, FAB, Profile Pic etc ?
3. I notice we still have the Back button in thereas well. @kazo0 I think we agreed we don't want that in there, correct me if I'm wrong :).

**Comment by iurycarlos** [comment]
1 - didn't get. You mean a Toggle button with image? No. Not yet. We should support icons with Image and that will affect all components. There's an issue about it.
2 - Flexibility is always interesting. However, right now, if i'm not wrong, our template would need adjustments to cover that.
3 - Did you noticed the message with the green screenshot is from may? It is not reflecting any recent work with themes. I just tagged you yesterday to let you follow the issue :)

**Comment by NVLudwig** [comment]
@iurycarlos 
1 - The expander/collapse button (burger menu icon) = Toggle
2 - Ok will review with Steve and may end up removing it.
3- Doh! I did not see the dates and thought you had solved the color seperation issues :(

**Comment by NVLudwig** [comment]
@iurycarlos  second Figma draft is ready for review
https://www.figma.com/file/trM3i5qUPmFBcnxWXXmnuF/branch/ewmJs0WraTrwmE9Sp8cvwE/Uno-Platform-Material-Toolkit?type=design&mode=design&t=s8VLlsSjK8pPDdsl-1

**Comment by NVLudwig** [comment]
@iurycarlos is this one done?

**Comment by iurycarlos** [comment]
Still need to be merged: https://github.com/unoplatform/Uno.Figma/pull/1509

**Comment by iurycarlos** [comment]
duplicated from https://github.com/unoplatform/Uno.Themes/issues/1219
